requirements.in

```diff
@@ -1,4 +1,4 @@
-hca-ingest~=2.5.2
+hca-ingest
```

Comment:
Don't we need a minimum of 2.5.2 for this project?

Reply:
in the [requirements.txt](https://github.com/ebi-ait/ingest-exporter/blob/b2891181687c240bd9b57628ab91b3115dd91312/requirements.txt#L83)
``` requirements.txt
hca-ingest==2.6.0
    # via -r requirements.in
```